src/generator/passes.jl

```diff
@@ -879,7 +881,7 @@ function (x::FunctionPrinter)(dag::ExprDAG, options::Dict)
     general_options = get(options, "general", Dict())
     log_options = get(general_options, "log", Dict())
     show_info = get(log_options, "FunctionPrinter_log", x.show_info)
-    ignorelist = get(general_options, "output_ignorelist", get(general_options, "printer_blacklist", []))
+    ignorelist = Regex.(get(general_options, "output_ignorelist", get(general_options, "printer_blacklist", [])))
```

Comment:
Better to do the type conversion in `should_exclude_node`, like:

```
for item ∈ ignorelist
      match(Regex(item), str_node) === nothing || return true
end
```

Reply:
Done.